blog/_posts/2020-05-26-create-your-own-theme-detailed-walkthrough.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Create your own theme in minutes !
+---
+
+In Convos [v4.xx](https://convos.chat/blog/2020/5/14/theming-support-in-4-point-oh.html) it is super easy to customize
+or create a theme.  Follow this detailed walkthrough where I create a **"hack3r"** theme ! 
+
+<!--more-->
+
+
+## Smooth start
+
+You have to connect to your server where convos is running or you can do this tutorial locally : 
+
+`cd $CONVOS_HOME/public/themes`
+
+Then copy past an existing theme :
+
+`cp high-contrast_color-scheme-light.css mytheme.css`
+
+The name of your file is not important, the theme name will be included in the header of the CSS file.
+
+`vi mytheme.css`
+
+Choose your theme name :
+
+``` /*
+ * name: Hacker */ ```
+
+When you go to your convos instance settings, you will already be able to select your new theme (maybe you will need to
+refresh your page) :
+
+![](/public/screenshots/2020-05-26-choosenewtheme.png)
+
+We created our theme, and moved on it. At the moment, our new theme is a vanilla copy of `high-contrast light` but it's
```

Comment:
What does "and moved on it" mean? Do you have an alternative sentence?

Reply:
switched